### PR TITLE
Add metadata section to return json in Retrieve governance connector, GET endpoint (Resolved Merge Conflict)

### PR DESCRIPTION
**Pull request by ravindudsilva**
## Purpose
Add metadata section to return json in ``Retrieve governance connector`` , GET endpoint. 

### Changes in this pull request
Updated openApi 3.0 definition in ``src/main/resources/identity-governance.yaml`` and regenerated the API of  `` org.wso2.carbon.identity.api.server.identity.governance.v1 `` and passed the metadata properties.

New Response
```
  "properties": [
        { "name": "Recovery.AdminPasswordReset.RecoveryLink",
            "value": "false",
            "displayName": "Enable password reset via recovery e-mail",
            "description": "User gets notified with a link to reset password",
            "meta": {
                "type": "boolean"
            }
       }
 ]
```

Related issue [10084](https://github.com/wso2/product-is/issues/10084)

* * *

New response after adding ``groupID``  to meta data section return JSON.

```       {
            "name": "SelfRegistration.CallbackRegex",
            "value": "[https://localhost:9443].*[/authenticationendpoint/login.do]",
            "displayName": "User self registration callback URL regex",
            "description": "This prefix will be used to validate the callback URL.",
            "meta": {
                "type": "string",
                "regex": ".*",
                "groupID": 0
            }
        },
```
* * *

* This PR is been made to fix the PR builder failure of [Original PR](https://github.com/wso2/identity-api-server/pull/275) 

* * * 
Tests
Following test scenarios were passed locally.
```
 <class name="org.wso2.identity.integration.test.rest.api.server.identity.governance.v1.IdentityGovernanceSuccessTest"/>
 <class name="org.wso2.identity.integration.test.rest.api.server.identity.governance.v1.IdentityGovernanceFailureTest"/>
```

### CODE REVIEW

**Comment by somindatommy** [comment]
@ravindudsilva shall we mention the tests that this PR was tested against?

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (somindatommy) [inline components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/MetaRes.java:2]: Shall we fix the license year?

File: components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/PropertyRes.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (somindatommy) [inline components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/gen/java/org/wso2/carbon/identity/api/server/identity/governance/v1/model/PropertyRes.java:2]: Lets not change the existing license

**Comment by ravindudsilva** [comment]
> @ravindudsilva shall we mention the tests that this PR was tested against?

https://github.com/wso2/product-is/tree/master/modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/identity/governance/v1

**Review by somindatommy — APPROVED** [review]

**Pull request merged**